website/client/src/components/tasks/task.vue

```diff
@@ -169,7 +169,6 @@
           >
             <div class="d-inline-flex">
               <div
-                v-if="isUser"
```

Comment:
`collapseChecklist()` makes a PUT request to `/api/v4/tasks/${task._id}` so I think it'll result in a 401 error for users not allowed to edit the group tasks

Reply:
Yeah, I think I ran into this during testing. Would receive a red snack notification.

Reply:
![image](https://user-images.githubusercontent.com/2686586/85312669-25707b00-b485-11ea-85d2-8f5c49f546a9.png)

Reply:
ok so what are our options:
- just let it save on the session not in the database? => no api call on the groups page
- we'd need a different place to store the combination of user, group and task to save the collapse state
- remove the toggle checklist (as it was) on the group page BUT without the space it had

did I miss anything?

Reply:
Maybe we can give all users in the group the permission to toggle the checklist? Having the option individually for each user would be quite complicate as not all users have a copy of the task

Reply:
Yeah, ALL users are still able to collapse checklists on tasks, just not complete them.

Reply:
@negue after talking with @Tressley the desired behavior is that the `collapsedChecklist` field of group tasks should be editable by everyone not just by the managers or the leader of the group but by all members

Reply:
ok, I didn't know there is also this kind of field :) , will enable it for all team users